src/rollup/types.d.ts

```diff
@@ -99,11 +100,7 @@ export type ResolveIdHook = (
 	parent: string
 ) => Promise<string | boolean | void | null> | string | boolean | void | null;
 
-export type IsExternal = (
-	id: string,
-	parentId: string,
-	isResolved: boolean
-) => Promise<boolean | void> | boolean | void;
+export type IsExternal = (id: string, parentId: string, isResolved: boolean) => boolean | void;
```

Comment:
Previously we said in the type that it would permit a Promise, but the implementation didn't actually handle the promise. So this actually corrects the type to match what happens.